File: Heat/ViewModels/Invoices/InsertedInvoicesGridViewModel.vb
Imports System.ComponentModel.DataAnnotations


Namespace ViewModels.Invoices
    ''' <summary>
    ''' Modello per la visualizzazione della griglia delle fatture inserite (ma non confermate).
    ''' </summary>
    ''' <remarks></remarks>
    Public Class InsertedInvoicesGridViewModel
        <Key> _
        Property ID As Integer

        <Display(name:="Cliente")> _
        Property Customer As String

        <Display(name:="Data emissione")> _
        <DisplayFormat(dataformatstring:="{0: d}")> _
        Property InvoiceDate As DateTime

        <Display(name:="Numero Provvisorio")> _
        Property InvoiceNumber As String

        <Display(name:="Numero righe")> _
        Property RowCount As Integer

    End Class

End Namespace

File: Heat.ConvertedToC#/ViewModels/Invoices/InsertedInvoicesGridViewModel.cs
using System;
using System.ComponentModel.DataAnnotations;


namespace Heat.ViewModels.Invoices
{
    /// <summary>
    /// Modello per la visualizzazione della griglia delle fatture inserite (ma non confermate).
    /// </summary>
    /// <remarks></remarks>
    public class InsertedInvoicesGridViewModel
	{
		[Key()]
		public int ID { get; set; }

		[Display(Name = "Cliente")]
		public string Customer { get; set; }

		[Display(Name = "Data emissione")]
		[DisplayFormat(DataFormatString = "{0: d}")]
		public DateTime InvoiceDate { get; set; }

		[Display(Name = "Numero Provvisorio")]
		public string InvoiceNumber { get; set; }

		[Display(Name = "Numero righe")]
		public int RowCount { get; set; }

	}

}
